past-workouts.php

```diff
@@ -203,8 +211,26 @@ function convertClimbTabletoWorkoutArray($climbTable) {
 			   iframe.src = 'downloadworkouts.php';
 			  
 			   document.body.appendChild(iframe);
-
 			};
+
+			$("#confirmWorkoutDeleteModal").on("shown.bs.modal", function(e) {
+				$(this).find('.btn-ok').data('workoutId', $(e.relatedTarget).data('workoutId'));
+			});
+
+			$('#confirmDeleteButton').click(function() {
+				var workoutId = $(this).data("workoutId");
+				$.ajax({
+					url: sitePath() + "/workout-delete?wid=" + workoutId,

```

Comment:
This was showing an ajax error for me. Fixed by adding ".php" to the path: url: sitePath() + "/workout-delete.php?wid=" + workoutId,